packages/loopring_v3/circuit/Circuits/WithdrawCircuit.h

```diff
@@ -39,7 +37,7 @@ class WithdrawCircuit : public BaseTransactionCircuit {
   DualVariableGadget nonce;
 
   // Signature
-  Poseidon_gadget_T<12, 1, 6, 53, 11, 1> hash;
+  Poseidon_gadget_T<10, 1, 6, 53, 9, 1> hash;
 
```

Comment:
Modify the Poseidon hash parameter to save constraint?

Reply:
Just less inputs to hash because a few were removed. A Poseidon hash is only like 300 constraints, not much too save on in any case for these single hashes. :)